servers/visual/shader_language.cpp

```diff
@@ -3769,8 +3769,7 @@ ShaderLanguage::Node *ShaderLanguage::_reduce_expression(BlockNode *p_block, Sha
 						nv.sint = -cn->values[i].sint;
 					} break;
 					case TYPE_UINT: {
-						// FIXME: This can't work on uint
-						nv.uint = -cn->values[i].uint;
+						nv.uint = 0 - cn->values[i].uint;
```

Comment:
Can you put a comment to state that we intentionally wrap around like GLSL does?

Reply:
Comment added.